focus-ios/Blockzilla/Onboarding/OnboardingFactory.swift

```diff
@@ -116,4 +145,44 @@ fileprivate extension String {
     static let defaultBrowserOnboardingViewSecondSubtitleV2 = String(format: NSLocalizedString("Onboarding.DefaultBrowser.SecondSubtitle.V2", value: "Make %@ your default to protect your data with every link you open.", comment: "Text for a label that indicates the second subtitle for the default browser onboarding screen version 2. %@ is the name of the app (Focus/Klar)"), AppInfo.shortProductName)
     static let defaultBrowserOnboardingViewTopButtonTitleV2 = NSLocalizedString("Onboarding.DefaultBrowser.TopButtonTitle.V2", value: "Set as Default Browser", comment: "Text for a label that indicates the title of the top button from the default browser onboarding screen version 2.")
     static let defaultBrowserOnboardingViewBottomButtonTitleV2 = NSLocalizedString("Onboarding.DefaultBrowser.BottomButtonTitle.V2", value: "Skip", comment: "Text for a label that indicates the title of the bottom button from the default browser onboarding screen version 2.")
+    static let termsOfUseTitle = NSLocalizedString(
+        "Onboarding.TermsOfUse.Title",
```

Comment:
@razvanlitianu 
This doesn't seem to work, I think you need to do the formatting elsewhere.

Xcode doesn't seem to be able to export the string correctly

```
      <trans-unit id="Onboarding.TermsOfUse.Title" xml:space="preserve">
        <source>Onboarding.TermsOfUse.Title</source>
        <note>Title for the Terms of Use screen during onboarding. %@ is the app name (for example “Firefox Focus” or ”Firefox Klar”).</note>
      </trans-unit>
```

This is now blocking exposing other strings
https://github.com/mozilla-l10n/focusios-l10n/pull/126

Reply:
I'm going to work around it by manually changing the string in the XLIFF to `Welcome to %@`; but this needs a proper fix here, otherwise the next import will try to undo the change.

The value needs to be just `value: "Welcome to %@"`